lib/image-inspector-client.rb

```diff
@@ -51,12 +51,7 @@ def fetch_metadata
     def exception_handler
       yield
     rescue RestClient::Exception => e
-      begin
-        json_error_msg = JSON.parse(e.response || '') || {}
-      rescue JSON::ParserError
-        json_error_msg = {}
-      end
-      err_message = json_error_msg['message'] || e.to_s
+      err_message = e.response || e.message || ''
```

Comment:
[This](https://github.com/rest-client/rest-client/blob/51cdf048ea534392a675661ea3902e4cf56bc808/lib/restclient/response.rb#L27) is why.

This bug started happening to us when they changed the to_s of another [class](https://github.com/rest-client/rest-client/commit/a5b0998fdcdd366bf35aff29dc1299faaabb790e#diff-eb43f38b5b854e38aa016f90e0ce30baR103) so I slightly prefer this way (which is a little more verbose)

I leave it to your consideration